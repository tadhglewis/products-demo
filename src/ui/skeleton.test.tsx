import { render } from '@testing-library/react';
import Skeleton from './skeleton';

describe('skeleton', () => {
  it('should match snapshot', async () => {
    const { container } = render(<Skeleton height={300} />);

    expect(container).toMatchSnapshot();
  });
});
