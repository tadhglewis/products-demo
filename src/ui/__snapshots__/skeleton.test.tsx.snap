// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`skeleton should match snapshot 1`] = `
.c0 {
  background-color: #f7f8f9;
  display: block;
  width: 100%;
  height: 300px;
  border-radius: 3px;
}

<div>
  <div
    class="c0"
    height="300"
  />
</div>
`;
